app/models/setting.rb

```diff
@@ -62,6 +62,7 @@ def validate(record)
   scope :organization_fact, -> { where('name not in (?)', 'organization_fact') unless Taxonomy.enabled_taxonomies.include? 'organizations' }
   scope :default_location, -> { where('name not in (?)', 'default_location') unless Taxonomy.enabled_taxonomies.include? 'locations' }
   scope :location_fact, -> { where('name not in (?)', 'location_fact') unless Taxonomy.enabled_taxonomies.include? 'locations' }
+  scope :order_by, ->(attr) { except(:order).order(attr) }
```

Comment:
The `except(:order)` is to remove `default_scope` https://github.com/swapab/foreman/blob/develop/app/models/setting.rb#L57
I feel the `default_scope` is not required and should be replaced with explicit sorting wherever required.